package.json

```diff
@@ -30,12 +30,13 @@
   },
   "homepage": "https://github.com/gvn/mofo-ui#readme",
   "peerDependencies": {
+    "rc-collapse": "1.6.3",
     "react": "15.0.1"
   },
   "dependencies": {
-    "rc-collapse": "1.6.3"
```

Comment:
I believe we do still want this here and not as a peerDependency (and science should remove `rc-collapse` as a direct dependency). Locally I removed rc-collapse and mofo-ui as dependencies of science, then `npm install`ed from package.json and from this branch and got errors.